representative/models.py

```diff
@@ -1283,8 +1291,9 @@ def update_or_create_representatives_are_not_duplicates(
         }
         return results
 
-    def update_representative_social_media(self, representative, representative_twitter_handle=False,
-                                      representative_facebook=False):
+    @staticmethod
+    def update_representative_social_media(representative, representative_twitter_handle=False,
```

Comment:
If all variables can't fit on one line, please break each one onto it's own line.